tests/foreman/api/test_role.py

```diff
@@ -112,6 +112,8 @@ def create_org_admin_role(self, name=None, orgs=None, locs=None):
                 'role': {'name': name, 'organization_ids': orgs or [], 'location_ids': locs or []}
             }
         )
+        if 'role' in org_admin:
+            return entities.Role(id=org_admin['role']['id']).read()
```

Comment:
Do we know why `role` key is present only sometimes? What does it depend on?

Reply:
If it's not consistent, it seems like either a product bug or Nailgun bug. Won't this fix just hide it?

Reply:
@mirekdlugosz @lhellebr sorry for delay, it is hard to reproduce this outside of our automation environment, though there it occurs regularly. Fwict the inconsistency is connected to multi-threaded run. Alternative candidate solution would be wrapping the class into @run_in_one_thread

Reply:
@pondrejk I think I would prefer `@run_in_one_thread` in this case. Seems to be cleaner solution.
I would like to wait for @lhellebr opinion, since he also commented. If Lukas is OK with current implementation, please go ahead and merge. I know these kind of failures are hard to track down, reproduce and fix.